src/containers/ClassifierContainer.jsx

```diff
@@ -28,10 +29,11 @@ class ClassifierContainer extends React.Component {
     this.useRotate90 = this.useRotate90.bind(this);
     this.useResetImage = this.useResetImage.bind(this);
     this.useContrast = this.useContrast.bind(this);
-  
+    this.showMetadata = this.showMetadata.bind(this);
+
     //TEMPORARY
     this.state = {
-      TEST_POPUP: null,
+      POPUP: null,
```

Comment:
Minor request: (Recommended for code consistency, but not blocking) change the all caps `POPUP` and `CLOSE_POPUP` names to standardCamelCaseNames to better match the rest of the code.

Apologies, I used the all caps naming originally to make sure the methods/variables related to the test popup were super obvious.